core/src/components/radio/radio.scss

```diff
@@ -59,3 +67,142 @@ input {
 :host(:focus) {
   outline: none;
 }
+
+:host(.in-item:not(.legacy-radio)) {
+  width: 100%;
+}
+
+// Radio Wrapper
+// --------------------------------------------------
+.radio-wrapper {
+  display: flex;
+
+  position: relative;
+
+  flex-grow: 1;
+
+  align-items: center;
+
+  height: 100%;
+
+  min-height: inherit;
+  
+  transition: background-color 15ms linear;
```

Comment:
This was added for MD inputs/textareas. Do we need this for the radio?

Reply:
Good callout, I don't think so. The radio wrapper doesn't have a background color directly. I'll remove and validate 👍